VAMobile/src/api/types/ClaimsAndAppealsData.ts

```diff
@@ -431,6 +431,10 @@ export type ClaimsAndAppealsList = {
     updatedAt: string
     displayTitle: string
     decisionLetterSent: boolean
+    phase?: number | string
```

Comment:
Could this type be narrowed to just a number? (optional)

Reply:
possibly, but it isn't harming anything being extra safe here without knowing for sure what the type is